packages/flutter/test/flutter_test_config.dart

```diff
@@ -41,7 +41,7 @@ Future<void> testExecutable(FutureOr<void> Function() testMain) {
   // receive the event.
   WidgetController.hitTestWarningShouldBeFatal = true;
 
-  if (_isLeakTrackingEnabled()) {
+  // if (_isLeakTrackingEnabled()) {
```

Comment:
Revert this.

Reply:
Oops ! Very sorry about that :( I reverted it in [refactor: Revert bad commit](https://github.com/flutter/flutter/pull/147403/commits/1f561cfb9d690df0f29bcf8a4486384069769495)